paasta_tools/cli/cmds/status.py

```diff
@@ -809,6 +817,47 @@ def _print_flink_status_from_job_manager(
         dashboard_url = metadata["annotations"].get("flink.yelp.com/dashboard_url")
         output.append(f"    URL: {dashboard_url}/")
 
+    cluster_without_pnw_dash = cluster.replace("pnw-", "")
+
+    # Print Flink config link resources
+    if verbose:
+        output.append(
+            f"    Yelpsoa configs: https://sourcegraph.yelpcorp.com/sysgit/yelpsoa-configs/-/tree/{service}"
+        )
+        output.append(
+            f"    Srv configs: https://sourcegraph.yelpcorp.com/sysgit/srv-configs/-/tree/ecosystem/"
+            f"{cluster_without_pnw_dash}/{service}"
+        )
+
+    # Print Flink Log Commands
+    if verbose:
```

Comment:
we have a bunch of `if verbose:` blocks in a row - should these be a single block?

Reply:
Possibly, it was more for just formatting/separating 
Tbf it's getting to the point where could maybe refactor whole function, will consider it in a new PR